File: src/kyu_7/over_the_road_1/python/over_the_road_1_test.py
import codewars_test as test
from solution import over_the_road
test.assert_equals(over_the_road(1, 3), 6)
test.assert_equals(over_the_road(3, 3), 4)
test.assert_equals(over_the_road(2, 3), 5)
test.assert_equals(over_the_road(3, 5), 8)
test.assert_equals(over_the_road(7, 11), 16)
test.assert_equals(over_the_road(23633656673, 310027696726), 596421736780)

File: src/kyu_7/over_the_road_1/csharp/over_the_road_1_test.cs
using NUnit.Framework;
using System;

[TestFixture]
public class Basic_Tests {
  [Test]
  public void basic_test_cases() {
    Assert.AreEqual(6, CodeWars.OverTheRoad(1, 3));
    Assert.AreEqual(4, CodeWars.OverTheRoad(3, 3));
    Assert.AreEqual(5, CodeWars.OverTheRoad(2, 3));
    Assert.AreEqual(8, CodeWars.OverTheRoad(3, 5));
    Assert.AreEqual(16, CodeWars.OverTheRoad(7, 11));
  }
}